### ISSUE TITLE
.NET Maui Custom Handlers not working, official docs is wrong?

### ISSUE THREAD

**Issue by runstackjp** [dotnet/maui#5659]
### Description

I am creating an application using .NET MAUI and would like to implement customization of the controls.
For this purpose, I have consulted the official MS documentation.
[https://docs.microsoft.com/en-us/dotnet/maui/user-interface/handlers/customize](https://docs.microsoft.com/en-us/dotnet/maui/user-interface/handlers/customize)

The problem is that I get a compile error.

- EntryMapper does not exist in EntryHandler.
- Colors.Red.ToNative() does not exist.
![image](https://user-images.githubusercontent.com/9915587/160762936-ff05468d-4a43-4de3-bdc6-67dbd1477bb3.png)

I am using translation software as I am not fluent in English. I am sorry if my words are wrong.

### Steps to Reproduce

1.First make a subclass of the Entry control:

```
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MauiAppSample
{
    public class MyEntry : Entry  
    {
    }
}
```

2.I then customize the EntryHandler to perform the desired modification to MyEntry instances:

```
namespace MauiAppSample;

public partial class App : Application
{
	public App()
	{
		InitializeComponent();

        Microsoft.Maui.Handlers.EntryHandler.EntryMapper.AppendToMapping(nameof(IView.Background), (handler, view) =>
        {
            if (view is MyEntry)
            {
#if ANDROID
                handler.NativeView.SetBackgroundColor(Colors.Red.ToNative());
#elif IOS
                  handler.NativeView.BackgroundColor = Colors.Red.ToNative();
                  handler.NativeView.BorderStyle = UIKit.UITextBorderStyle.Line;
#elif WINDOWS
                  handler.NativeView.Background = Colors.Red.ToNative();
#endif
            }
        });
    }
}
```

### Version with bug

Preview 14 (current)

### Last version that worked well

Unknown/Other

### Affected platforms

I was *not* able test on other platforms

### Affected platform versions

all

### Did you find any workaround?

There is no workaround.

**Comment by Mephisztoe** [comment]
Please have a look at the Preview 14 Migration Guide. There have been some changes in regards to platform specific implementations. You can find more about that [here](https://github.com/dotnet/maui/wiki/Migrating-to-Preview-14).

In your case:
Rename EntryMapper to just Mapper and NativeView to PlatformView. It should work then.

I added this as an issue in the docs repo [here](https://github.com/dotnet/docs-maui/issues/396).

**Comment by runstackjp** [comment]
@Mephisztoe Thank you for your answer I can't find ToAndroid, but I was able to confirm that it works by using Android.Graphics.Color instead.
![image](https://user-images.githubusercontent.com/9915587/160766416-13cf491a-b860-4fed-8544-e2827f4098f5.png)

**Comment by runstackjp** [comment]
Using Microsoft.Maui.Controls.Compatibility.Platform.Android;
was missing.
The ToAndroid issue has been resolved.

**Comment by Mephisztoe** [comment]
Right. I was going to say the same. In my App.xaml.cs I've got additionally:

```
#if __ANDROID__
using Android.Content.Res;
using Microsoft.Maui.Controls.Compatibility.Platform.Android;
#endif
```

**Comment by runstackjp** [comment]
Thank you. Problem solved.
